frappe/public/js/lib/leaflet_easy_button/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2014 Daniel Montague
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
Can we update the attributions file (in project root) to include any more vendored or not?

I noticed that leaflet* projects are not mentioned there. There might be more or unused projects mentioned too 😅

Reply:
Tried this automated attributions file generation, but it felt too verbose so skipped it. 
<img width="1196" alt="Screenshot 2022-07-04 at 4 04 37 PM" src="https://user-images.githubusercontent.com/13928957/177148621-b5b1ba5b-b1d6-40be-9279-a7329ba38649.png">

---

Manually updated attributions file for now [a24af60](https://github.com/frappe/frappe/pull/17289/commits/a24af60f037e558f183ee6c7966c9aa0e2c6c7fa). Just mentioned vendored packages and icons.